acra/guides/advanced-integrations/client-side-integration-with-acra-connector.md

```diff
@@ -0,0 +1,224 @@
+
+## Client side: AcraConnector and AcraWriter
+
+### AcraConnector and AcraWriter
+
+AcraConnector is a (separate) service running alongside your application — it pretends to be a database listener, relays all the requests to AcraServer, receives the responses, and returns them to the app, just like a normal database listener would do.
+
+To talk to AcraServer, you'll need to run AcraConnector on the same host as your application, in a separate container or as a separate user. You'll also need to route database requests to its address.
+
+To talk to AcraTranslator you'll need to do the same: run AcraConnector on the same host as your application, in a separate container or as a separate user, and use its URL as destination URL from your application.
+
+#### Why do we need a special piece of software to talk to your other piece of software?
+
+Acra needs a trusted agent on the application's side to protect the sensitive decrypted responses, to provide basic channel authentication, and to enforce certain behaviours.
+
+AcraConnector acts as a local proxy that receives requests from the app and returns decrypted answers. AcraConnector provides an encrypted and authenticated connection to AcraServer, which, in turn, fetches the response to the request from the database and decrypts all the data. AcraServer then returns the data to AcraConnector via a secure connection.
+
+AcraConnector works in a similar fashion with AcraTranslator, redirecting AcraStructs from application to AcraTranslator and delivering the decrypted response back.
+
+This enforces maximum secrecy, employing authentication that is easy to manage (pre-shared private keys), and requires minimal intervention into your code for successful implementation!
+
+
+### Getting started with AcraConnector
+
+#### Method 1A. Launch AcraConnector using Docker (the fastest way to try AcraConnector)
+> Note: Using Docker is recommended for testing purposes only. Please don't rely on Docker in real-life production settings.    
+> Note: The following examples focus on using AcraConnector and AcraWriter with PostgreSQL, but Acra also supports MySQL.
+
+Clone the Acra repository, build images, and start Docker compose with PostgreSQL, AcraServer, AcraConnector, and Secure Session between them:
+
+```
+git clone https://github.com/cossacklabs/acra.git
+make docker
+docker-compose -f docker-compose.pgsql-nossl-server-ssession-connector.yml up
+```
+
+After running this command, the basic infrastructure is all set up, all components are connected, and the keys are distributed into the appropriate folder.
+
+Now, proceed to step [launching AcraConnector]({{< ref "acra/configuring-maintaining/general-configuration/acra-connector.md#launching-acraconnector-INVALID" >}}).
+Or install AcraConnector manually.
+
+#### Method 1B. Manual launch
+
+> Note: Skip this if you've used the Docker method described above.
+
+- Install dependencies: [Themis]({{< ref "/themis/installation/" >}}) cryptographic library:
```

Comment:
it is already this link. `{{< ref "/themis/installation/" >}}` will expand to `/themis/installation/`. And it's an absolute link without a domain. So on `https://docs.cossacklabs.com` it will be https://docs.cossacklabs.com/themis/installation/.